docs/content/en/docs/Advanced/bundles.md

```diff
@@ -0,0 +1,149 @@
+
+- *Package*: This type is a [luet](https://luet.io) package that will be installed in the system. It requires the luet repository definition in order to work. Luet packages are a powerful way to manage dependencies and install software on your system.
+
+Note: In the future, Kairos will also support a local type for use in airgap situations, where you can pre-add bundles to the image before deployment.
+
+It's important to note that bundles do not have any special meaning in terms of immutability. They install files over paths that are mutable in the system, as they are simply overlaid during the boot process. This means that you can use bundles to make changes to your system at any time, even after it has been deployed.
+
+
+## Create bundles
+
+To build your own bundle, you will need to create a Dockerfile and any necessary files and scripts. A bundle is simply a container image that includes all the necessary assets to perform a specific task.
+
+To create a bundle, you will need to define a base image and copy over any necessary files and scripts to the image. For example, you might use the following Dockerfile to create a bundle image that deploys everything inside `assets` in the Kubernetes cluster:
+
+```Dockerfile
+FROM alpine
+COPY ./run.sh /
+COPY ./assets /assets
+```
+
+And the associated `run.sh` that installs the assets depending on a cloud-config keyword can be:
+
+```bash
+#!/bin/bash
+
+K3S_MANIFEST_DIR="/var/lib/rancher/k3s/server/manifests/"
+
+mkdir -p $K3S_MANIFEST_DIR
+
+# IF the user sets `example.enable` in the input cloud config, we install our assets
+if [ "$(kairos-agent config get example.enable | tr -d '\n')" == "true" ]; then
+  cp -rf assets/* $K3S_MANIFEST_DIR
+fi
+```
+
+This Dockerfile creates an image based on the Alpine base image, and copies over a script file and some assets to the image. 
+You can then add any additional instructions to the Dockerfile to install necessary packages, set environment variables, or perform any other tasks required by your bundle.
+
+Once you have created your Dockerfile and any necessary script files, you can build your bundle image by running docker build and specifying the path to your Dockerfile. 
+
+For example:
+
+```bash
+docker build -t <image> .
+```
+
+This command will build an image with the name "my-bundle" based on the instructions in your Dockerfile.
+
+After building your bundle image, you will need to push it to a registry so that it can be accessed by Kairos. You can use a public registry like Docker Hub. To push your image to a registry, use the docker push command. For example:
+
+```bash
+docker push <image>
+```
+
+This will push the "my-bundle" image to your specified registry.
+
+And use it with Kairos:
+
+```yaml
+#cloud-config
+
+bundles:
+- targets:
+  - run://<image>
```

Comment:
the `<image>`  used here is the same as the tag in 123 but if I understand correctly here i need the full path e.g. `quay.io/kairos/community-bundles:metallb_latest`